kusd/validator/states.go

```diff
@@ -236,7 +246,7 @@ func (val *Validator) commitState() stateFn {
 	voter, err := val.network.IsVoter(&bind.CallOpts{}, val.account.Address)
 	if err != nil {
 		// @TODO (rgeraldes) - complete
-		//log.Error()
+		log.Crit("Failed to verify if the validator is a voter")
```

Comment:
Should we `return nil` here?

Reply:
We need to study these situations carefully (related to the punishment issue).